backend/src/server/createServer.ts

```diff
@@ -84,6 +85,10 @@ export const createServer = async (
 
   app.get('/hook/gms/' + CONFIG.GMS_WEBHOOK_SECRET, gmsWebhook)
 
+  // OpenID Connect
+  app.get('/.well-known/openid-configuration', openidConfiguration)
+  app.get('/.well-known/jwks.json', jwks)
```

Comment:
As i understand the OpendID-Connect documentation with keycloak the endpoint to get the publickeys is predefined too (`/realms/{realm-name}/protocol/openid-connect/certs`)

please see https://www.keycloak.org/securing-apps/oidc-layers#_certificate_endpoint